packages/plugins/telemetry/package.json

```diff
@@ -0,0 +1,38 @@
+{
+    "name": "@dd/telemetry-plugins",
+    "packageManager": "yarn@4.0.2",
+    "license": "MIT",
+    "private": true,
+    "author": "Datadog",
+    "exports": {
+        ".": "./src/index.ts",
+        "./esbuild-plugin/*": "./src/esbuild-plugin/*.ts",
+        "./webpack-plugin/*": "./src/webpack-plugin/*.ts",
+        "./common/*": "./src/common/*.ts",
+        "./common/metrics/*": "./src/common/metrics/*.ts",
+        "./common/output/*": "./src/common/output/*.ts",
+        "./*": "./src/*.ts"
+    },
```

Comment:
You'll need a `publishConfig` otherwise those won't be valid for the built code

Also isn't `"./**": "./src/**.ts"` already working for all those options? (except `.`)

And you may want to add `./package.json` (common to include)

Reply:
Most of the publication work is done in https://github.com/DataDog/build-plugin/pull/65